manifests/Insecure/Npcap/0.9991.yaml

```diff
@@ -0,0 +1,17 @@
+Id: Insecure.Npcap
+Version: 0.9991
+Name: Npcap
+Publisher: Insecure
+License: Modified GNU GPLv2
+LicenseUrl: https://nmap.org/book/man-legal.html
+Tags: network, security
+Description: Npcap is the Nmap Project's packet sniffing (and sending) library for Windows.
+Homepage: https://nmap.org/npcap/
+Installers:
+  - Arch: x86
+    Url: https://nmap.org/npcap/dist/npcap-0.9991.exe
+    Sha256: FF00C9B15073FB26C3CEA7F2DB4A3DDA4306FC0D49B66CA703CE0B36B72E9BD4
+    InstallerType: exe
+    Switches:
+      Silent: "/S"
```

Comment:
**Error:** Silent installation is only supported in [Npcap OEM](https://nmap.org/npcap/oem/)

Reply:
Removed.